RecoLocalMuon/GEMSegment/plugins/GEMSegmentBuilder.cc

```diff
@@ -47,9 +47,9 @@ void GEMSegmentBuilder::build(const GEMRecHitCollection* recHits, GEMSegmentColl
     // this reference id serves to link all GEMEtaPartitions
     // and will also be used to determine the GEMSuperChamber 
     // to which the GEMSegment is assigned (done inside GEMSegAlgoXX)
-    GEMDetId id(it2->gemId().region(),1,it2->gemId().station(),0,it2->gemId().chamber(),0);
+    //GEMDetId id(it2->gemId().region(),1,it2->gemId().station(),0,it2->gemId().chamber(),0);
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

If the "id" is no longer needed, please also cleanup the comments above it which describe why the ```named "id"``` is needed.